azurerm/internal/services/eventgrid/eventgrid_event_subscription_resource.go

```diff
@@ -91,6 +92,30 @@ func resourceArmEventGridEventSubscription() *schema.Resource {
 				Computed: true,
 			},
 
+			"azure_function_endpoint": {
```

Comment:
could we add an acceptance test covering this new block?

Reply:
I would really like to do an acceptance test for it. Unfortunately, it's pretty much the same as the webhook endpoint, which is also hard to test. You would first have to be able to deploy a function with an input binding of type `eventGridBinding` out of Terraform. But that is currently not possible as there is no terraform resource for it. It is only possible to deploy a function app, which hosts multiple functions inside.

I started with an implementation of an `azurerm_function` resource to also be able to write acceptance tests for the function and webhook endpoints of the `eventgrid_event_subscription`, but it's not that easy to find a good schema design for it.